tests/backstop_data/scenarios/manage-cases.json

```diff
@@ -86,6 +86,25 @@
       "url": "{url}/civicrm/case/a/#/case/list?caseId={caseId}&tab=Files",
       "onReadyScript": "manage-cases/files-tab-upload-file.js"
     },
+    {
+      "label": "Manage Cases - Case Overview - Linked Cases Panel",
```

Comment:
This should not be a new Scenario, this should be merged with `Manage Cases - Case Overview`. Because we just take one screenshot when `civicaseAllowLinkedCasesTab` is False.